lib/app_form_builder.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AppFormBuilder < ActionView::Helpers::FormBuilder
```

Comment:
The fact that you could do this and then specify a `builder` for the form is new to me. Sweet!

Reply:
Yeah. You can also, in your app config do something like

```ruby
ActionView::Base.default_form_builder = AppFormBuilder
```

In the past, I've used gems for Bootstrap or Foundation that do this behind the scenes.